### PR TITLE
Make empty string default for altText prop

### PR DESCRIPTION
**Pull request by megcrenshaw**
After listening to just how bad our images sound when there is no alt text provided ...

https://user-images.githubusercontent.com/9142121/197048380-c3d6ad65-25e7-4c7c-b3c9-b88e54b3e76b.mov

I've made the empty string the default alt text for the components I could find that have images. Are there others you know of? Now the default sounds like this:

https://user-images.githubusercontent.com/9142121/197048676-d572504e-cd7b-4937-9ca8-aa5554851b8c.mov

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/VerticalImageResourceCard.jsx [hunk 1]
```diff
@@ -84,7 +86,7 @@ class VerticalImageResourceCard extends Component {
       <div style={cardStyle}>
         <div style={imageStyle}>
           <a href={link}>
-            <img src={imgSrc} alt={title} />
+            <img src={imgSrc} alt={altText} />
```
Comment (megcrenshaw) [inline apps/src/templates/VerticalImageResourceCard.jsx:89]: This is a behavior change. I think that's okay, as the `title` is read within this component anyways.

Comment (epeach) [reply apps/src/templates/VerticalImageResourceCard.jsx:89]: Agreed. A net positive behavior change.

**Review by epeach — APPROVED** [review]
Love this! I think it sets us up for success in a lot of ways.

**Comment by TurnerRiley** [comment]
Thank you for doing this! These changes look good to me!

**Review by mgc1194 — APPROVED** [review]

**Pull request merged**